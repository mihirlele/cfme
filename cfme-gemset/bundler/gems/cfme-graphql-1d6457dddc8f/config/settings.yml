---
:graphql_api:
  :authentication_timeout: 30.seconds
