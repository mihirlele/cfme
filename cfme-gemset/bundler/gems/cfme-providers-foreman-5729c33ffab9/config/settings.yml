---
:ems:
  :ems_foreman:
    :blacklisted_event_names: []
    :event_handling:
      :event_groups:
:http_proxy:
  :foreman:
    :host:
    :password:
    :port:
    :user:
