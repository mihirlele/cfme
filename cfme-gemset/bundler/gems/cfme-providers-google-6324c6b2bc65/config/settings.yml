---
:ems:
  :ems_google:
    :blacklisted_event_names: []
    :event_handling:
      :event_groups:
:ems_refresh:
  :gce:
    :inventory_object_refresh: true
    :inventory_collections:
      :saver_strategy: default
  :gce_network:
    :inventory_object_refresh: true
    :inventory_collections:
      :saver_strategy: default
:http_proxy:
  :google:
    :host:
    :password:
    :port:
    :user:
:workers:
  :worker_base:
    :event_catcher:
      :event_catcher_google:
        :poll: 15.seconds
    :queue_worker_base:
      :ems_metrics_collector_worker:
        :ems_metrics_collector_worker_google: {}
      :ems_refresh_worker:
        :ems_refresh_worker_google: {}
        :ems_refresh_worker_google_network: {}
