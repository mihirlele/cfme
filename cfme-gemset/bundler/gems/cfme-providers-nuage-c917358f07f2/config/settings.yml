---
:ems:
  :ems_nuage:
    :blacklisted_event_names: []
    :event_handling:
      :event_groups:
        :addition:
          :critical:
          - !ruby/regexp /^nuage_.+_create$/
        :deletion:
          :critical:
          - !ruby/regexp /^nuage_.+_delete$/
        :update:
          :critical:
          - !ruby/regexp /^nuage_.+_update$/
        :status: # alarms
          :critical:
          - !ruby/regexp /^nuage_alarm_.+$/
:http_proxy:
  :nuage:
    :host:
    :password:
    :port:
    :user:
:ems_refresh:
  :nuage_network:
    :inventory_object_refresh: true
    :allow_targeted_refresh: true
:workers:
  :worker_base:
    :event_catcher:
      :event_catcher_nuage_network:
        :topics:
          - topic/CNAMessages
          - topic/CNAAlarms
        :amqp_connect_timeout: 5.seconds
