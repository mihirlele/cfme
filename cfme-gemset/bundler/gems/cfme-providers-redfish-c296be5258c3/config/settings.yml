---
:ems:
  :ems_redfish:
    :blacklisted_event_names: []
    :event_handling:
      :event_groups:
        :general:
          :critical:
          - !ruby/regexp /^redfish_/

:ems_refresh:
  :redfish_ph_infra:
    :inventory_object_refresh: true

:http_proxy:
  :redfish:
    :host:
    :password:
    :port:
    :user:

:log:
  :level_redfish: info

:workers:
  :worker_base:
    :queue_worker_base:
      :ems_refresh_worker:
        :ems_refresh_worker_redfish_physical_infra: {}
    :event_catcher:
      :event_catcher_redfish: {}
