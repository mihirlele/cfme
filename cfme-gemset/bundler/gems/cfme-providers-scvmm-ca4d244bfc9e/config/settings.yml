---
:ems:
  :ems_scvmm:
    :blacklisted_event_names: []
    :event_handling:
      :event_groups:
:ems_refresh:
  :scvmm:
    :inventory_object_refresh: false
:http_proxy:
  :scvmm:
    :host:
    :password:
    :port:
    :user:
