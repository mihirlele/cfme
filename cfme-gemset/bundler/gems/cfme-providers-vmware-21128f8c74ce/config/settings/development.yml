---
:development:
  :use_vim_broker_in_console: false
