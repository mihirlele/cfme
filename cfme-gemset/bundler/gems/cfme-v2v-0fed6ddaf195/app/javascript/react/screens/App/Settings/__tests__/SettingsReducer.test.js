import settingsReducer, { initialState } from '../SettingsReducer';
import {
  V2V_FETCH_SERVERS,
  V2V_FETCH_SETTINGS,
  V2V_PATCH_SETTINGS,
  FETCH_V2V_CONVERSION_HOSTS,
  SHOW_V2V_CONVERSION_HOST_WIZARD,
  HIDE_V2V_CONVERSION_HOST_WIZARD
} from '../SettingsConstants';
import { servers, settings } from '../settings.fixures';

it('sets default state', () => {
  const action = { type: '@@INIT' };
  const state = settingsReducer(undefined, action);
  expect(state).toMatchSnapshot();
});

describe('fetching servers', () => {
  it('is pending', () => {
    const action = {
      type: `${V2V_FETCH_SERVERS}_PENDING`
    };
    const prevState = initialState.set('fetchingServersRejected', true);
    const state = settingsReducer(prevState, action);
    expect(state).toMatchSnapshot();
  });

  it('is rejected', () => {
    const action = {
      type: `${V2V_FETCH_SERVERS}_REJECTED`,
      payload: 'error'
    };
    const prevState = initialState.set('isFetchingServers', true);
    const state = settingsReducer(prevState, action);
    expect(state).toMatchSnapshot();
  });

  it('is successful', () => {
    const action = {
      type: `${V2V_FETCH_SERVERS}_FULFILLED`,
      payload: { data: servers }
    };
    const prevState = initialState.set('fetchingServersRejected', true).set('isFetchingServers', true);
    const state = settingsReducer(prevState, action);
    expect(state.fetchingServersRejected).toBe(false);
    expect(state.isFetchingServers).toBe(false);
    expect(state.servers).toHaveLength(2);
  });
});

describe('fetching settings', () => {
  it('is pending', () => {
    const action = {
      type: `${V2V_FETCH_SETTINGS}_PENDING`
    };
    const prevState = initialState.set('fetchingSettingsRejected', true);
    const state = settingsReducer(prevState, action);
    expect(state).toMatchSnapshot();
  });

  it('is rejected', () => {
    const action = {
      type: `${V2V_FETCH_SETTINGS}_REJECTED`,
      payload: 'error'
    };
    const prevState = initialState.set('isFetchingSettings', true);
    const state = settingsReducer(prevState, action);
    expect(state).toMatchSnapshot();
  });

  it('is successful', () => {
    const action = {
      type: `${V2V_FETCH_SETTINGS}_FULFILLED`,
      payload: { data: settings }
    };
    const prevState = initialState.set('fetchingSettingsRejected', true).set('isFetchingSettings', true);
    const state = settingsReducer(prevState, action);
    expect(state).toMatchSnapshot();
  });
});

describe('saving settings', () => {
  it('is pending', () => {
    const action = {
      type: `${V2V_PATCH_SETTINGS}_PENDING`
    };
    const prevState = initialState.set('savingSettingsRejected', true);
    const state = settingsReducer(prevState, action);
    expect(state).toMatchSnapshot();
  });

  it('is rejected', () => {
    const action = {
      type: `${V2V_PATCH_SETTINGS}_REJECTED`,
      payload: 'error'
    };
    const prevState = initialState.set('isSavingSettings', true);
    const state = settingsReducer(prevState, action);
    expect(state).toMatchSnapshot();
  });

  it('is successful', () => {
    const action = {
      type: `${V2V_PATCH_SETTINGS}_FULFILLED`,
      payload: settings
    };
    const prevState = initialState.set('savingSettingsRejected', true).set('isSavingSettings', true);
    const state = settingsReducer(prevState, action);
    expect(state).toMatchSnapshot();
  });
});

describe('fetching conversion hosts', () => {
  it('is pending', () => {
    const action = {
      type: `${FETCH_V2V_CONVERSION_HOSTS}_PENDING`
    };
    const prevState = initialState.set('isRejectedFetchingConversionHosts', true);
    const state = settingsReducer(prevState, action);
    expect(state).toMatchSnapshot();
  });

  it('is rejected', () => {
    const action = {
      type: `${FETCH_V2V_CONVERSION_HOSTS}_REJECTED`,
      payload: 'error'
    };
    const prevState = initialState.set('isFetchingConversionHosts', true);
    const state = settingsReducer(prevState, action);
    expect(state).toMatchSnapshot();
  });

  it('is successful', () => {
    const action = {
      type: `${FETCH_V2V_CONVERSION_HOSTS}_FULFILLED`,
      payload: { data: { resources: [{ mock: 'conversionHost' }] } }
    };
    const prevState = initialState
      .set('isRejectedFetchingConversionHosts', true)
      .set('isFetchingConversionHosts', true);
    const state = settingsReducer(prevState, action);
    expect(state).toMatchSnapshot();
  });
});

describe('showing and hiding the conversion host wizard', () => {
  it('shows', () => {
    const action = { type: SHOW_V2V_CONVERSION_HOST_WIZARD };
    const state = settingsReducer(initialState, action);
    expect(state.conversionHostWizardVisible).toBeTruthy();
  });

  it('hides', () => {
    const action = { type: HIDE_V2V_CONVERSION_HOST_WIZARD };
    const prevState = initialState.set('conversionHostWizardVisible', true);
    const state = settingsReducer(prevState, action);
    expect(state.conversionHostWizardVisible).toBeFalsy();
  });
});
