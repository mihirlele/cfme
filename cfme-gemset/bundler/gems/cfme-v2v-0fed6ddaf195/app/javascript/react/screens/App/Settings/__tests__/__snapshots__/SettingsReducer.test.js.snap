// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fetching conversion hosts is pending 1`] = `
Object {
  "conversionHostDeleteModalVisible": false,
  "conversionHostRetryModalMounted": false,
  "conversionHostRetryModalVisible": false,
  "conversionHostTaskToRetry": null,
  "conversionHostTasks": Array [],
  "conversionHostTasksByResource": Object {},
  "conversionHostToDelete": null,
  "conversionHostWizardMounted": false,
  "conversionHostWizardVisible": false,
  "conversionHosts": Array [],
  "errorDeleteConversionHost": false,
  "errorFetchingConversionHostTasks": null,
  "errorFetchingConversionHosts": null,
  "errorFetchingServers": null,
  "errorFetchingSettings": null,
  "errorPostingConversionHosts": null,
  "errorSavingSettings": null,
  "fetchingServersRejected": false,
  "fetchingSettingsRejected": false,
  "isDeletingConversionHost": false,
  "isFetchingConversionHostTasks": false,
  "isFetchingConversionHosts": true,
  "isFetchingServers": false,
  "isFetchingSettings": false,
  "isPostingConversionHosts": false,
  "isRejectedDeletingConversionHost": false,
  "isRejectedFetchingConversionHostTasks": false,
  "isRejectedFetchingConversionHosts": false,
  "isRejectedPostingConversionHosts": false,
  "isSavingSettings": false,
  "postConversionHostsResults": Array [],
  "savedSettings": Object {},
  "savingSettingsRejected": false,
  "servers": Array [],
}
`;

exports[`fetching conversion hosts is rejected 1`] = `
Object {
  "conversionHostDeleteModalVisible": false,
  "conversionHostRetryModalMounted": false,
  "conversionHostRetryModalVisible": false,
  "conversionHostTaskToRetry": null,
  "conversionHostTasks": Array [],
  "conversionHostTasksByResource": Object {},
  "conversionHostToDelete": null,
  "conversionHostWizardMounted": false,
  "conversionHostWizardVisible": false,
  "conversionHosts": Array [],
  "errorDeleteConversionHost": false,
  "errorFetchingConversionHostTasks": null,
  "errorFetchingConversionHosts": "error",
  "errorFetchingServers": null,
  "errorFetchingSettings": null,
  "errorPostingConversionHosts": null,
  "errorSavingSettings": null,
  "fetchingServersRejected": false,
  "fetchingSettingsRejected": false,
  "isDeletingConversionHost": false,
  "isFetchingConversionHostTasks": false,
  "isFetchingConversionHosts": false,
  "isFetchingServers": false,
  "isFetchingSettings": false,
  "isPostingConversionHosts": false,
  "isRejectedDeletingConversionHost": false,
  "isRejectedFetchingConversionHostTasks": false,
  "isRejectedFetchingConversionHosts": true,
  "isRejectedPostingConversionHosts": false,
  "isSavingSettings": false,
  "postConversionHostsResults": Array [],
  "savedSettings": Object {},
  "savingSettingsRejected": false,
  "servers": Array [],
}
`;

exports[`fetching conversion hosts is successful 1`] = `
Object {
  "conversionHostDeleteModalVisible": false,
  "conversionHostRetryModalMounted": false,
  "conversionHostRetryModalVisible": false,
  "conversionHostTaskToRetry": null,
  "conversionHostTasks": Array [],
  "conversionHostTasksByResource": Object {},
  "conversionHostToDelete": null,
  "conversionHostWizardMounted": false,
  "conversionHostWizardVisible": false,
  "conversionHosts": Array [
    Object {
      "mock": "conversionHost",
    },
  ],
  "errorDeleteConversionHost": false,
  "errorFetchingConversionHostTasks": null,
  "errorFetchingConversionHosts": null,
  "errorFetchingServers": null,
  "errorFetchingSettings": null,
  "errorPostingConversionHosts": null,
  "errorSavingSettings": null,
  "fetchingServersRejected": false,
  "fetchingSettingsRejected": false,
  "isDeletingConversionHost": false,
  "isFetchingConversionHostTasks": false,
  "isFetchingConversionHosts": false,
  "isFetchingServers": false,
  "isFetchingSettings": false,
  "isPostingConversionHosts": false,
  "isRejectedDeletingConversionHost": false,
  "isRejectedFetchingConversionHostTasks": false,
  "isRejectedFetchingConversionHosts": false,
  "isRejectedPostingConversionHosts": false,
  "isSavingSettings": false,
  "postConversionHostsResults": Array [],
  "savedSettings": Object {},
  "savingSettingsRejected": false,
  "servers": Array [],
}
`;

exports[`fetching servers is pending 1`] = `
Object {
  "conversionHostDeleteModalVisible": false,
  "conversionHostRetryModalMounted": false,
  "conversionHostRetryModalVisible": false,
  "conversionHostTaskToRetry": null,
  "conversionHostTasks": Array [],
  "conversionHostTasksByResource": Object {},
  "conversionHostToDelete": null,
  "conversionHostWizardMounted": false,
  "conversionHostWizardVisible": false,
  "conversionHosts": Array [],
  "errorDeleteConversionHost": false,
  "errorFetchingConversionHostTasks": null,
  "errorFetchingConversionHosts": null,
  "errorFetchingServers": null,
  "errorFetchingSettings": null,
  "errorPostingConversionHosts": null,
  "errorSavingSettings": null,
  "fetchingServersRejected": false,
  "fetchingSettingsRejected": false,
  "isDeletingConversionHost": false,
  "isFetchingConversionHostTasks": false,
  "isFetchingConversionHosts": false,
  "isFetchingServers": true,
  "isFetchingSettings": false,
  "isPostingConversionHosts": false,
  "isRejectedDeletingConversionHost": false,
  "isRejectedFetchingConversionHostTasks": false,
  "isRejectedFetchingConversionHosts": false,
  "isRejectedPostingConversionHosts": false,
  "isSavingSettings": false,
  "postConversionHostsResults": Array [],
  "savedSettings": Object {},
  "savingSettingsRejected": false,
  "servers": Array [],
}
`;

exports[`fetching servers is rejected 1`] = `
Object {
  "conversionHostDeleteModalVisible": false,
  "conversionHostRetryModalMounted": false,
  "conversionHostRetryModalVisible": false,
  "conversionHostTaskToRetry": null,
  "conversionHostTasks": Array [],
  "conversionHostTasksByResource": Object {},
  "conversionHostToDelete": null,
  "conversionHostWizardMounted": false,
  "conversionHostWizardVisible": false,
  "conversionHosts": Array [],
  "errorDeleteConversionHost": false,
  "errorFetchingConversionHostTasks": null,
  "errorFetchingConversionHosts": null,
  "errorFetchingServers": "error",
  "errorFetchingSettings": null,
  "errorPostingConversionHosts": null,
  "errorSavingSettings": null,
  "fetchingServersRejected": true,
  "fetchingSettingsRejected": false,
  "isDeletingConversionHost": false,
  "isFetchingConversionHostTasks": false,
  "isFetchingConversionHosts": false,
  "isFetchingServers": false,
  "isFetchingSettings": false,
  "isPostingConversionHosts": false,
  "isRejectedDeletingConversionHost": false,
  "isRejectedFetchingConversionHostTasks": false,
  "isRejectedFetchingConversionHosts": false,
  "isRejectedPostingConversionHosts": false,
  "isSavingSettings": false,
  "postConversionHostsResults": Array [],
  "savedSettings": Object {},
  "savingSettingsRejected": false,
  "servers": Array [],
}
`;

exports[`fetching settings is pending 1`] = `
Object {
  "conversionHostDeleteModalVisible": false,
  "conversionHostRetryModalMounted": false,
  "conversionHostRetryModalVisible": false,
  "conversionHostTaskToRetry": null,
  "conversionHostTasks": Array [],
  "conversionHostTasksByResource": Object {},
  "conversionHostToDelete": null,
  "conversionHostWizardMounted": false,
  "conversionHostWizardVisible": false,
  "conversionHosts": Array [],
  "errorDeleteConversionHost": false,
  "errorFetchingConversionHostTasks": null,
  "errorFetchingConversionHosts": null,
  "errorFetchingServers": null,
  "errorFetchingSettings": null,
  "errorPostingConversionHosts": null,
  "errorSavingSettings": null,
  "fetchingServersRejected": false,
  "fetchingSettingsRejected": false,
  "isDeletingConversionHost": false,
  "isFetchingConversionHostTasks": false,
  "isFetchingConversionHosts": false,
  "isFetchingServers": false,
  "isFetchingSettings": true,
  "isPostingConversionHosts": false,
  "isRejectedDeletingConversionHost": false,
  "isRejectedFetchingConversionHostTasks": false,
  "isRejectedFetchingConversionHosts": false,
  "isRejectedPostingConversionHosts": false,
  "isSavingSettings": false,
  "postConversionHostsResults": Array [],
  "savedSettings": Object {},
  "savingSettingsRejected": false,
  "servers": Array [],
}
`;

exports[`fetching settings is rejected 1`] = `
Object {
  "conversionHostDeleteModalVisible": false,
  "conversionHostRetryModalMounted": false,
  "conversionHostRetryModalVisible": false,
  "conversionHostTaskToRetry": null,
  "conversionHostTasks": Array [],
  "conversionHostTasksByResource": Object {},
  "conversionHostToDelete": null,
  "conversionHostWizardMounted": false,
  "conversionHostWizardVisible": false,
  "conversionHosts": Array [],
  "errorDeleteConversionHost": false,
  "errorFetchingConversionHostTasks": null,
  "errorFetchingConversionHosts": null,
  "errorFetchingServers": null,
  "errorFetchingSettings": "error",
  "errorPostingConversionHosts": null,
  "errorSavingSettings": null,
  "fetchingServersRejected": false,
  "fetchingSettingsRejected": true,
  "isDeletingConversionHost": false,
  "isFetchingConversionHostTasks": false,
  "isFetchingConversionHosts": false,
  "isFetchingServers": false,
  "isFetchingSettings": false,
  "isPostingConversionHosts": false,
  "isRejectedDeletingConversionHost": false,
  "isRejectedFetchingConversionHostTasks": false,
  "isRejectedFetchingConversionHosts": false,
  "isRejectedPostingConversionHosts": false,
  "isSavingSettings": false,
  "postConversionHostsResults": Array [],
  "savedSettings": Object {},
  "savingSettingsRejected": false,
  "servers": Array [],
}
`;

exports[`fetching settings is successful 1`] = `
Object {
  "conversionHostDeleteModalVisible": false,
  "conversionHostRetryModalMounted": false,
  "conversionHostRetryModalVisible": false,
  "conversionHostTaskToRetry": null,
  "conversionHostTasks": Array [],
  "conversionHostTasksByResource": Object {},
  "conversionHostToDelete": null,
  "conversionHostWizardMounted": false,
  "conversionHostWizardVisible": false,
  "conversionHosts": Array [],
  "errorDeleteConversionHost": false,
  "errorFetchingConversionHostTasks": null,
  "errorFetchingConversionHosts": null,
  "errorFetchingServers": null,
  "errorFetchingSettings": null,
  "errorPostingConversionHosts": null,
  "errorSavingSettings": null,
  "fetchingServersRejected": false,
  "fetchingSettingsRejected": false,
  "isDeletingConversionHost": false,
  "isFetchingConversionHostTasks": false,
  "isFetchingConversionHosts": false,
  "isFetchingServers": false,
  "isFetchingSettings": false,
  "isPostingConversionHosts": false,
  "isRejectedDeletingConversionHost": false,
  "isRejectedFetchingConversionHostTasks": false,
  "isRejectedFetchingConversionHosts": false,
  "isRejectedPostingConversionHosts": false,
  "isSavingSettings": false,
  "postConversionHostsResults": Array [],
  "savedSettings": Object {
    "max_concurrent_tasks_per_ems": 10,
    "max_concurrent_tasks_per_host": 10,
  },
  "savingSettingsRejected": false,
  "servers": Array [],
}
`;

exports[`saving settings is pending 1`] = `
Object {
  "conversionHostDeleteModalVisible": false,
  "conversionHostRetryModalMounted": false,
  "conversionHostRetryModalVisible": false,
  "conversionHostTaskToRetry": null,
  "conversionHostTasks": Array [],
  "conversionHostTasksByResource": Object {},
  "conversionHostToDelete": null,
  "conversionHostWizardMounted": false,
  "conversionHostWizardVisible": false,
  "conversionHosts": Array [],
  "errorDeleteConversionHost": false,
  "errorFetchingConversionHostTasks": null,
  "errorFetchingConversionHosts": null,
  "errorFetchingServers": null,
  "errorFetchingSettings": null,
  "errorPostingConversionHosts": null,
  "errorSavingSettings": null,
  "fetchingServersRejected": false,
  "fetchingSettingsRejected": false,
  "isDeletingConversionHost": false,
  "isFetchingConversionHostTasks": false,
  "isFetchingConversionHosts": false,
  "isFetchingServers": false,
  "isFetchingSettings": false,
  "isPostingConversionHosts": false,
  "isRejectedDeletingConversionHost": false,
  "isRejectedFetchingConversionHostTasks": false,
  "isRejectedFetchingConversionHosts": false,
  "isRejectedPostingConversionHosts": false,
  "isSavingSettings": true,
  "postConversionHostsResults": Array [],
  "savedSettings": Object {},
  "savingSettingsRejected": false,
  "servers": Array [],
}
`;

exports[`saving settings is rejected 1`] = `
Object {
  "conversionHostDeleteModalVisible": false,
  "conversionHostRetryModalMounted": false,
  "conversionHostRetryModalVisible": false,
  "conversionHostTaskToRetry": null,
  "conversionHostTasks": Array [],
  "conversionHostTasksByResource": Object {},
  "conversionHostToDelete": null,
  "conversionHostWizardMounted": false,
  "conversionHostWizardVisible": false,
  "conversionHosts": Array [],
  "errorDeleteConversionHost": false,
  "errorFetchingConversionHostTasks": null,
  "errorFetchingConversionHosts": null,
  "errorFetchingServers": null,
  "errorFetchingSettings": null,
  "errorPostingConversionHosts": null,
  "errorSavingSettings": "error",
  "fetchingServersRejected": false,
  "fetchingSettingsRejected": false,
  "isDeletingConversionHost": false,
  "isFetchingConversionHostTasks": false,
  "isFetchingConversionHosts": false,
  "isFetchingServers": false,
  "isFetchingSettings": false,
  "isPostingConversionHosts": false,
  "isRejectedDeletingConversionHost": false,
  "isRejectedFetchingConversionHostTasks": false,
  "isRejectedFetchingConversionHosts": false,
  "isRejectedPostingConversionHosts": false,
  "isSavingSettings": false,
  "postConversionHostsResults": Array [],
  "savedSettings": Object {},
  "savingSettingsRejected": true,
  "servers": Array [],
}
`;

exports[`saving settings is successful 1`] = `
Object {
  "conversionHostDeleteModalVisible": false,
  "conversionHostRetryModalMounted": false,
  "conversionHostRetryModalVisible": false,
  "conversionHostTaskToRetry": null,
  "conversionHostTasks": Array [],
  "conversionHostTasksByResource": Object {},
  "conversionHostToDelete": null,
  "conversionHostWizardMounted": false,
  "conversionHostWizardVisible": false,
  "conversionHosts": Array [],
  "errorDeleteConversionHost": false,
  "errorFetchingConversionHostTasks": null,
  "errorFetchingConversionHosts": null,
  "errorFetchingServers": null,
  "errorFetchingSettings": null,
  "errorPostingConversionHosts": null,
  "errorSavingSettings": null,
  "fetchingServersRejected": false,
  "fetchingSettingsRejected": false,
  "isDeletingConversionHost": false,
  "isFetchingConversionHostTasks": false,
  "isFetchingConversionHosts": false,
  "isFetchingServers": false,
  "isFetchingSettings": false,
  "isPostingConversionHosts": false,
  "isRejectedDeletingConversionHost": false,
  "isRejectedFetchingConversionHostTasks": false,
  "isRejectedFetchingConversionHosts": false,
  "isRejectedPostingConversionHosts": false,
  "isSavingSettings": false,
  "postConversionHostsResults": Array [],
  "savedSettings": Object {
    "max_concurrent_tasks_per_ems": 10,
    "max_concurrent_tasks_per_host": 10,
  },
  "savingSettingsRejected": false,
  "servers": Array [],
}
`;

exports[`sets default state 1`] = `
Object {
  "conversionHostDeleteModalVisible": false,
  "conversionHostRetryModalMounted": false,
  "conversionHostRetryModalVisible": false,
  "conversionHostTaskToRetry": null,
  "conversionHostTasks": Array [],
  "conversionHostTasksByResource": Object {},
  "conversionHostToDelete": null,
  "conversionHostWizardMounted": false,
  "conversionHostWizardVisible": false,
  "conversionHosts": Array [],
  "errorDeleteConversionHost": false,
  "errorFetchingConversionHostTasks": null,
  "errorFetchingConversionHosts": null,
  "errorFetchingServers": null,
  "errorFetchingSettings": null,
  "errorPostingConversionHosts": null,
  "errorSavingSettings": null,
  "fetchingServersRejected": false,
  "fetchingSettingsRejected": false,
  "isDeletingConversionHost": false,
  "isFetchingConversionHostTasks": false,
  "isFetchingConversionHosts": false,
  "isFetchingServers": false,
  "isFetchingSettings": false,
  "isPostingConversionHosts": false,
  "isRejectedDeletingConversionHost": false,
  "isRejectedFetchingConversionHostTasks": false,
  "isRejectedFetchingConversionHosts": false,
  "isRejectedPostingConversionHosts": false,
  "isSavingSettings": false,
  "postConversionHostsResults": Array [],
  "savedSettings": Object {},
  "savingSettingsRejected": false,
  "servers": Array [],
}
`;
